Source/Hurl.Settings/Views/Dialogs/TestRules.xaml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="Hurl.Settings.Views.Dialogs.TestRules"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Hurl.Settings.Views.Dialogs"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkit="using:CommunityToolkit.WinUI.Controls"
+    mc:Ignorable="d">
+
+    <ScrollViewer
+        Padding="8,0"
+        HorizontalAlignment="Center"
+        ScrollViewer.VerticalScrollBarVisibility="Auto">
+        <StackPanel Width="480" MinHeight="430">
+            <toolkit:SettingsCard Header="Enter a URI to test">
+                <TextBox
+                    x:Name="_UriInput"
+                    MinWidth="120"
+                    MaxWidth="300" />
+            </toolkit:SettingsCard>
+
+            <toolkit:SettingsCard Margin="0,8,0,0" Header="Test against Existing Rules">
```

Comment:
This part isn't really a Setting. I would use only the Button here.

Reply:
I think I would have to come back and tackle the Test Rules UI later on. I don't like how it looks/works right now.